src/decompiler-ast.h

```diff
@@ -226,6 +226,12 @@ struct AST {
             mc.GetLabel(cast<BrIfExpr>(&e)->var)->label_type;
         return;
       }
+      // https://github.com/WebAssembly/wabt/issues/1922
```

Comment:
While its useful for the review, I don't think we need to annotate all these locations with bug urls.

It would be great to see tests added though.

Normally for santizer failures one need to construct a test under `test/binary/` containing the module the triggers the bad code path.   Since the tests run under santizers it should be possible to verify that the tests fail before this change and pass afterwards.